modules/ROOT/pages/tokenization.adoc

```diff
@@ -20,15 +20,22 @@ The Tokenization service provides the following benefits:
 ** Protection of sensitive information reduces your sensitive data zones.
```

Comment:
Is it possible to rewrite the items in this list so the sentences have roughly the same construction? We go from verb phrase (sentence fragment) to noun phrases and it's a little jarring. Ignore this comment if changes would result in artificially twisting the sentences. The goal is ease of reading, we can live with a little inconsistent structure if that's the easiest read.